### ISSUE TITLE
Error in vqcache

### ISSUE THREAD

**Issue by GoogleCodeExporter** [rohitrng/vqmod#126]
```
NOTE THAT THIS IS FOR VQMOD ENGINE ERRORS ONLY. FOR GENERAL ERRORS FROM
MODIFICATIONS CONTACT YOUR DEVELOPER

What steps will reproduce the problem?
1. viewing order in admin panel

What is the expected output? What do you see instead?

order detail

Notice: Undefined variable: vqmod in 
/home/onlinec3/public_html/vqmod/vqcache/vq2-system_engine_controller.php on 
line 56
Fatal error: Call to a member function modCheck() on a non-object in 
/home/onlinec3/public_html/vqmod/vqcache/vq2-system_engine_controller.php on 
line 56

vQmod Version:2.3.2
Server Operating System: php server 5.3.27
Please provide any additional information below.

```

Original issue reported on code.google.com by `prateekg...@gmail.com` on 20 Aug 2013 at 3:49

**Comment by GoogleCodeExporter** [comment]
```
I fixed the issue just by upgrading vqmod to 2.4.1. Still a great thanx for 
your support truly great.You can close this issue. :)
```

Original comment by `prateekg...@gmail.com` on 20 Aug 2013 at 4:17

**Comment by GoogleCodeExporter** [comment]
Original comment by `DJG6...@gmail.com` on 20 Aug 2013 at 5:25
- Changed state: **Invalid**
